secant.xcodeproj/project.pbxproj

```diff
@@ -2301,7 +2301,7 @@
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_ENTITLEMENTS = "zashi-internal.entitlements";
 				CODE_SIGN_STYLE = Automatic;
-				CURRENT_PROJECT_VERSION = 3;
```

Comment:
what does this constant mean btw?

Reply:
iOS has 2 values, version and build. Version 1.0.1 for example, build 1. Build 2 would mean same version 1.0.1 but next build in a row. CURRENT_PROJECT_VERSION represents that build. The build is reseted to 1 with every version bump. So 1.0.2 would start from build 1 again.